hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -123,6 +123,7 @@ class KiaUvoApiEU(ApiImpl):
 
     def __init__(self, region: int, brand: int, language: str) -> None:
         # Users were complaining about the warning message.   Stating it is already english.  The below handles this but still throws warnings for non english items.
+        self.ccu_ccs2_protocol_support = None
```

Comment:
This should be moved up a line.  As that comment is explaining why it looks for the en and coverts variants to EN.

Reply:
Done. Thanks for the review.